### PR TITLE
Request to Merge  Smashing Magazine Feature Branch to Main

### PR DESCRIPTION
**Pull request by SunilDevalla**
This project aims to create a Heat Map of the page SmashingMagazine.com, The page works best on a 1366x768 resolution and using the Chrome browser.

### CODE REVIEW

File: index.html
Comment (jubaan) [inline index.html]: - [x] The links should be inside a `<ul>` tag and the links wrapped around `<li>` tags

File: index.html
Comment (jubaan) [inline index.html]: - [x] The links should be inside a `<ul>` tag and the links wrapped around `<li>` tags

File: index.html
Comment (jubaan) [inline index.html]: - [x] The links should be inside a `<ul>` tag and the links wrapped around `<li>` tags

File: style.css
Comment (jubaan) [inline style.css]: - [x] `border-radius: 5px`, `border-radius: 10px` is ussed repeated times in the file, it would be better to create something like the next selectors and apply the selector as a class in each element that uses that kind of border radious. This will help to maintain the code DRY.
```css
br-5 {
   border-radius: 5px;
}

br-10 {
  border-radius: 10px;
}
```

**Review by jubaan — CHANGES_REQUESTED** [review]
# Status: Changes Required :recycle: 

## :wave: :wave: Hello @ . Thanks for submitting you project!

Great thing this project has and has done well:
:100: The project has the main sections
:100:

There are some issues that need to be addressed before we can move forward. Please review the list below.

## Required changes :wrench: 
### [Mandatory] :red_circle: 

- [x] The links should be inside a `<ul>` tag and the links wrapped around `<li>` tags. See inline comments.

- [x] Section is not visually identifiable. It mentions the image and the text but it should stand out as you did with other elements
image

- [x] Maintain CSS file DRY (don't repeat yourself). Look for repeated properties and create utility selectors. See inline comments. And use this idea to eliminate repeadet code thought the CSS file.

- [x] Change your stylesheet path to match this syntax `./`
```
// Bad Paths
<link rel="stylesheet" href="/assets/stylesheets/main.css">
<link rel="stylesheet" href="/assets/js/bootstrap.min.js">
<img src="/imgs/myimage.png">


// Good Paths
<link rel="stylesheet" href="./assets/stylesheets/main.css">
<link rel="stylesheet" href="./assets/js/bootstrap.min.js">
<img src="./imgs/myimage.png">
```

- [x] The branch name should be descriptive and meaningful. 
```
// Bad name

Development

// Improved name

smashin-magazine-heatmap
```

### [OPTIONAL]  :yellow_circle:  

- [ ] @DsunilK Twitter account doesn't exist, please use a valid twitter account.


## Keep up the great work :muscle: and Happy Coding :smiley: :computer:!!

File: index.html
Comment (SunilDevalla) [reply index.html]: Thanks for the suggestions, made the changes accordingly.

File: index.html
Comment (SunilDevalla) [reply index.html]: Thanks for the above suggestions.

File: index.html
Comment (SunilDevalla) [reply index.html]: Thanks for the suggestions, made the changes accordingly.

File: style.css
Comment (SunilDevalla) [reply style.css]: Thanks for the suggestions, made the changes accordingly.

**Review by PhillipUg — APPROVED** [review]
# Status: Project Approved! 🥇 
Congratulations on making all the requested changes and getting this project approved. :+1:

You can merge this branch into the main branch when you're ready and go on to your next project.
:bulb:  Do not forget to submit your project completion form after merging this branch.

All the best team,

For any assistance, you can contact me on slack as **Phillip Musiime**

**Pull request merged**
